tests/integration/conftest.py

```diff
@@ -370,3 +421,73 @@ def is_mysql_ready():
         ),
     )
     await ops_test.model.wait_for_idle()
+
+
+@pytest.fixture(scope="module", name="test_image")
+def image_fixture():
+    """A PNG image that can be used in tests."""
+    image_response = requests.get(
+        "https://s.w.org/style/images/about/WordPress-logotype-wmark.png", timeout=10
```

Comment:
Should we vendor this? As we move towards GitHub self-hosted runners HTTP access to the external world will be reduced and vending would also mean that if, for some reason, that image is removed, our tests don't break

Reply:
I am a little bit concerned about the copyright and trademark issue if we include this image in our project.

Reply:
Can we use another image or does it have to be this one? E.g., just pick something in the public domain?

Reply:
Yes, let's use a Canonical logo instead.